coordinator/src/cli.rs

```diff
@@ -34,4 +40,40 @@ impl Opts {
 
         Ok(data_dir)
     }
+
+    /// Returns a list of addresses under which the node can be reached. Note this is used for the
+    /// node announcements.
+    pub fn p2p_announcement_addresses(&self) -> Vec<NetAddress> {
+        let mut addresses: Vec<NetAddress> = vec![];
+        if !self.p2p_address.ip().is_unspecified() {
```

Comment:
I believe this works for a local setup but not for production where we are either behind a NAT or inside a docker container.

Reply:
yes, I thought so too.. so, lets add it when we need it. 😉